performance_tests/requirements_mpl.txt

```diff
@@ -0,0 +1 @@
+amazon-cryptographic-material-providers-test-vectors @ git+https://github.com/aws/aws-cryptographic-material-providers-library.git@lucmcdon/python-mpl#subdirectory=TestVectorsAwsCryptographicMaterialProviders/runtimes/python
```

Comment:
Does this need the testvectors? Can you just use the regular MPL?

Reply:
Yes, regular mpl should be good! Thanks for pointing this out.